lib/client.go

```diff
@@ -138,6 +143,18 @@ func (c *Client) Init() error {
 			return err
 		}
 
+		idemixCurveIDOverride := os.Getenv("IDEMIX_CURVE_ID")
```

Comment:
I don't think there is a precedent for such environment variables, can you use a CLI flag instead? Similar to the ones at:
https://github.com/hyperledger/fabric-ca/blob/main/docs/source/clientcli.rst

Reply:
fixed